### ISSUE TITLE
Image completion is incorrectly using batch norm in training mode

### ISSUE THREAD

**Issue by bamos** [bamos/dcgan-completion.tensorflow#3]
A minor bug, the easiest fix is to use the sampler instead of the generator, which sets `train=False` in the batch norm. A better solution would be to use `tflearn`'s batch normalization code with their `is_training` variable so that there doesn't need to be a generator and sampler.
